com.xycz.simple_live.yaml

```diff
@@ -0,0 +1,279 @@
+  # - --system-talk-name=this.is.a.dummy.bus.name
+  - --system-talk-name=org.freedesktop.NetworkManager.IP4Config
+modules:
+  - name: simple_live
+    buildsystem: simple
+    build-commands:
+      # copy lib and data folders
+      - install -d "${FLATPAK_DEST}/lib"
+      - cp -r lib/* "${FLATPAK_DEST}/lib/"
+      - install -d "${FLATPAK_DEST}/data"
+      - cp -r data/* "${FLATPAK_DEST}/data/"
+
+      # copy executable
+      - install -Dm755 simple_live_app "${FLATPAK_DEST}/simple_live_app"
+      - mkdir -p "${FLATPAK_DEST}/bin"
+      - ln -s "${FLATPAK_DEST}/simple_live_app" "${FLATPAK_DEST}/bin/simple_live"
+
+      # copy metadata files
+      - install -Dm644 com.xycz.simple_live.desktop "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm644 com.xycz.simple_live.metainfo.xml "${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+      - install -Dm644 com.xycz.simple_live_512.png "${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png"
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        strip-components: 0
+        # Link needs to be replaced when this is upstreamed
+        url: https://github.com/s-kk/dart_simple_live/releases/download/v1.7.6/simple_live_app-1.7.5+10705-linux.zip
+        sha256: a03445da66e25da89199ebb598abe1badf2d5aa5056cd3ee3efe5c6bd43259bd
+      - type: file
+        path: com.xycz.simple_live.metainfo.xml
+      - type: file
+        path: com.xycz.simple_live.desktop
+      - type: file
+        dest-filename: com.xycz.simple_live_512.png
+        url: https://raw.githubusercontent.com/s-kk/dart_simple_live/692380b8c44ccaeafb2f9697b224e2617f6abe64/simple_live_app/assets/logo_circle_512.png
+        sha256: d9b4c02f2c0efe6485f6071050fae255b8074aa170950ecc90e74c9514c69f13
+    modules:
+      - name: libmpv
+        cleanup:
+          - /include
+          - /lib/pkgconfig
+          - /share/applications
+          - /share/bash-completion
+          - /share/doc
+          - /share/icons
+          - /share/man
+          - /share/zsh
+        buildsystem: meson
+        config-opts:
+          - -Dlibmpv=true
+          - -Dbuild-date=false
+          - -Dalsa=disabled
+          - -Dlua=disabled
+          - -Dcplayer=false
+        sources:
+          - type: archive
+            url: https://github.com/mpv-player/mpv/archive/v0.39.0.tar.gz
+            sha256: 2ca92437affb62c2b559b4419ea4785c70d023590500e8a52e95ea3ab4554683
+            x-checker-data:
+              type: anitya
+              project-id: 5348
+              stable-only: true
+              url-template: https://github.com/mpv-player/mpv/archive/v$version.tar.gz
+              is-important: true
+
+        modules:
+
+          - name: libv4l2
+            buildsystem: meson
+            cleanup:
+              - /sbin
+              - /bin
+              - /include
+              - /lib/pkgconfig
+              - /share/man
+              - /share/doc
```

Comment:
make these a top level cleanup so that you don't have to repeat for everything.

Reply:
Now moved shared cleanups to the top-level cleanup.